File: src/Omnia.ProcessManagement.Models/Processes/ProcessData.cs
using Omnia.Fx.Models.Language;
using Omnia.ProcessManagement.Models.CanvasDefinitions;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.Processes
{
    public class ProcessData
    {
        public ProcessData() 
        {
            this.Content = new MultilingualString();
        }
        public CanvasDefinition CanvasDefinition { get; set; }
        public MultilingualString Content { get; set; }
        public object DocumentBlockData { get; set; } //TODO
        public List<Link> Links { get; set; }
        public List<Task> Tasks { get; set; }

        public virtual string CreatedBy { get; set; }
        public virtual string ModifiedBy { get; set; }
        public virtual DateTimeOffset CreatedAt { get; set; }
        public virtual DateTimeOffset ModifiedAt { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processes/ProcessData.ts
import { CanvasDefinition } from '../drawingdefinitions';
import { MultilingualString } from '@omnia/fx-models';
import { Link } from './Link';
import { Task } from './Task';
import { DocumentRollupBlockData } from '@omnia/dm/models';

export interface ProcessData {
    canvasDefinition: CanvasDefinition;
    content: MultilingualString;

    documentBlockData?: DocumentRollupBlockData;
    links: Array<Link>; //TODO
    tasks: Array<Task>; //TODO


    createdBy: string;
    modifiedBy: string;
    createAt: Date;
    modifiedAt: Date;
}